### PR TITLE
UIDATIMP-38: Create UI for styling for `Completed` file uploads

### PR DESCRIPTION
**Pull request by VictorSoroka1**
## Purpose

Create UI for styling for Completed file uploads

## Approach

Extend `FileItem` component to support uploaded and deleting file uploading states. Additionally, cover error case when file removal was not succeded. In this case, the state should be changed back to the uploaded state.

### CODE REVIEW

File: src/components/UploadingJobsDisplay/UploadingJobsDisplay.js [hunk 1]
```diff
@@ -1,62 +1,88 @@
...
-  createFileDefinition, // eslint-disable-line
-  prepareFilesToUpload, // eslint-disable-line
-  uploadFiles,
-} from './utils/upload';
+import createUrl from '../../utils/createUrl';
+import * as API from './utils/upload';
+
+const DEFAULT_TIMEOUT_BEFORE_FILE_DELETION = 0;
+
+const NoUploadedFilesMessage = () => (
```
Comment (maximdidenkoepam) [inline src/components/UploadingJobsDisplay/UploadingJobsDisplay.js]: This function is used once, can we move the implementation to the place of the invocation?

File: src/components/UploadingJobsDisplay/components/FileItem/FileItem.js [hunk 2]
```diff
@@ -3,27 +3,30 @@ import React, {
...
-  UPLOADING,
-  FAILED,
-} from './fileItemStatuses';
+import * as fileStatuses from './fileItemStatuses';
 import Progress from '../../../Progress';
 
 import css from './FileItem.css';
 
-const getFileItemMeta = (props) => {
+const getFileItemMeta = props => {
```
Comment (maximdidenkoepam) [inline src/components/UploadingJobsDisplay/components/FileItem/FileItem.js]: I suggest to move getFileItemMeta from the file to another place, and then import it and use in the component

File: src/components/UploadingJobsDisplay/utils/upload.js [hunk 3]
```diff
@@ -119,3 +119,29 @@ export const uploadFiles = (
...
+    const error = {
+      status,
+      statusText,
+    };
+
+    throw new Error(error);
+  }
+};
+
+export const deleteFile = (url, headers) => {
```
Comment (maximdidenkoepam) [inline src/components/UploadingJobsDisplay/utils/upload.js]: Can we use async/await here?

File: src/components/UploadingJobsDisplay/UploadingJobsDisplay.js [hunk 4]
```diff
@@ -127,19 +165,58 @@ class UploadingJobsDisplay extends Component {
...
+          console.error(error); // eslint-disable-line no-console
+        });
+    }, timeoutBeforeFileDeletion);
+
+    this.updateFileState(file, { status: fileStatuses.DELETING });
+  };
+
+  deleteFileFromState = key => {
+    const { files } = this.state;
+    const updatedFiles = Object.assign({}, files);
```
Comment (OleksandrYehorovEPAM) [inline src/components/UploadingJobsDisplay/UploadingJobsDisplay.js]: Consider rewriting with spread operator instead
```suggestion
    const updatedFiles = { ...files };
```

File: src/components/UploadingJobsDisplay/utils/upload.js (hunk 3 above)
Comment (VictorSoroka1) [reply src/components/UploadingJobsDisplay/utils/upload.js]: I would keep using as is as this approach is used in other places in this file. Moreover, we have not decided on strictly using async/await syntax.

File: src/components/UploadingJobsDisplay/components/FileItem/FileItem.js (hunk 2 above)
Comment (VictorSoroka1) [reply src/components/UploadingJobsDisplay/components/FileItem/FileItem.js]: Makes sense. Will do.

File: src/components/UploadingJobsDisplay/UploadingJobsDisplay.js (hunk 1 above)
Comment (VictorSoroka1) [reply src/components/UploadingJobsDisplay/UploadingJobsDisplay.js]: I thought about it. I think it makes sense eventually, so will do this.

File: src/components/Progress/Progress.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ const progressInfoFormatters = {
   messagedPercentage: (current, total, payload) => (
     <React.Fragment>
```
Comment (OleksandrYehorovEPAM) [inline src/components/Progress/Progress.js]: Consider using named import for Fragment
```suggestion
    <Fragment>
          // ...
    </Fragment>
```

File: src/components/UploadingJobsDisplay/components/FileItem/FileItem.js [hunk 6]
```diff
@@ -3,27 +3,30 @@ import React, {
   Fragment,
 } from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import {
+  FormattedMessage,
+  FormattedDate,
+} from 'react-intl';
 import classNames from 'classnames';
+import noop from 'lodash/noop';
```
Comment (OleksandrYehorovEPAM) [inline src/components/UploadingJobsDisplay/components/FileItem/FileItem.js]: Let's be consistent and use named import.
```suggestion
import { noop } from 'lodash';
```

File: src/components/Progress/Progress.js (hunk 5 above)
Comment (VictorSoroka1) [reply src/components/Progress/Progress.js]: Agree with that.

File: src/components/UploadingJobsDisplay/components/FileItem/FileItem.js (hunk 6 above)
Comment (VictorSoroka1) [reply src/components/UploadingJobsDisplay/components/FileItem/FileItem.js]: Will do.

**Pull request merged**